docs/ko/docs/help-fastapi.md

```diff
@@ -1,162 +1,269 @@
-* # FastAPI 지원 - 도움말 받기
+# FastAPI 지원 - 도움 받기
 
-  **FastAPI** 가 마음에 드시나요?
+**FastAPI** 가 마음에 드시나요?
 
-  FastAPI, 다른 사용자, 개발자를 응원하고 싶으신가요?
+FastAPI, 다른 사용자, 개발자를 응원하고 싶으신가요?
 
-  혹은 **FastAPI** 에 대해 도움이 필요하신가요?
+혹은 **FastAPI** 에 대해 도움이 필요하신가요?
 
-  아주 간단하게 응원할 수 있습니다 (몇 번의 클릭만으로).
+아주 간단하게 응원할 수 있습니다 (몇 번의 클릭만으로).
 
-  또한 도움을 받을 수 있는 방법도 몇 가지 있습니다.
+또한 도움을 받을 수 있는 방법도 몇 가지 있습니다.
 
-  ## 뉴스레터 구독
+## 뉴스레터 구독
 
-  [**FastAPI와 친구** 뉴스레터](https://github.com/fastapi/fastapi/blob/master/newsletter)를 구독하여 최신 정보를 유지할 수 있습니다{.internal-link target=_blank}:
+[**FastAPI and friends** 뉴스레터](newsletter.md){.internal-link target=\_blank}를 구독하여 최신 정보를 유지할 수 있습니다:
 
-  - FastAPI 와 그 친구들에 대한 뉴스 🚀
-  - 가이드 📝
-  - 특징 ✨
-  - 획기적인 변화 🚨
-  - 팁과 요령 ✅
+* FastAPI and friends에 대한 뉴스 🚀
+* 가이드 📝
+* 기능 ✨
+* 획기적인 변화 🚨
+* 팁과 요령 ✅
 
-  ## 트위터에서 FastAPI 팔로우하기
+## 트위터에서 FastAPI 팔로우하기
 
-  [Follow @fastapi on **Twitter**](https://twitter.com/fastapi) 를 팔로우하여 **FastAPI** 에 대한 최신 뉴스를 얻을 수 있습니다. 🐦
+<a href="https://twitter.com/fastapi" class="external-link" target="_blank">**Twitter**의 @fastapi를 팔로우</a>하여 **FastAPI** 에 대한 최신 뉴스를 얻을 수 있습니다. 🐦
 
-  ## Star **FastAPI** in GitHub
+## Star **FastAPI** in GitHub
 
-  GitHub에서 FastAPI에 "star"를 붙일 수 있습니다(오른쪽 상단의 star 버튼을 클릭): https://github.com/fastapi/fastapi. ⭐️
+GitHub에서 FastAPI에 "star"를 붙일 수 있습니다 (오른쪽 상단의 star 버튼을 클릭): <a href="https://github.com/fastapi/fastapi" class="external-link" target="_blank">https://github.com/fastapi/fastapi</a>. ⭐️
 
-  스타를 늘림으로써, 다른 사용자들이 좀 더 쉽게 찾을 수 있고, 많은 사람들에게 유용한 것임을 나타낼 수 있습니다.
+스타를 늘림으로써, 다른 사용자들이 좀 더 쉽게 찾을 수 있고, 많은 사람들에게 유용한 것임을 나타낼 수 있습니다.
 
-  ## GitHub 저장소에서  릴리즈 확인
+## GitHub 저장소에서 릴리즈 확인
 
-  GitHub에서 FastAPI를 "watch"할 수 있습니다 (오른쪽 상단 watch 버튼을 클릭): https://github.com/fastapi/fastapi. 👀
+GitHub에서 FastAPI를 "watch"할 수 있습니다 (오른쪽 상단 watch 버튼을 클릭): <a href="https://github.com/fastapi/fastapi" class="external-link" target="_blank">https://github.com/fastapi/fastapi</a>. 👀
 
-  여기서 "Releases only"을 선택할 수 있습니다.
+여기서 "Releases only"을 선택할 수 있습니다.
 
-  이렇게하면, **FastAPI** 의 버그 수정 및 새로운 기능의 구현 등의 새로운 자료 (최신 버전)이 있을 때마다 (이메일) 통지를 받을 수 있습니다.
+이렇게하면, **FastAPI** 의 버그 수정 및 새로운 기능의 구현 등의 새로운 자료 (최신 버전)이 있을 때마다 (이메일) 통지를 받을 수 있습니다.
 
-  ## 개발자와의 연결
+## 개발자와의 연결
 
-  개발자인 [me (Sebastián Ramírez / `tiangolo`)](https://tiangolo.com/) 와 연락을 취할 수 있습니다.
+개발자인 <a href="https://tiangolo.com" class="external-link" target="_blank">me (Sebastián Ramírez / `tiangolo`)</a>와 연락을 취할 수 있습니다.
```

Comment:
```suggestion
<a href="https://tiangolo.com" class="external-link" target="_blank">개발자(Sebastián Ramírez / `tiangolo`)</a>와 연락을 취할 수 있습니다.
```
I think it would be better if you modify it to "개발자와 연락을" or "개발자인 저와 연락을"